qchem/pennylane_qchem/qchem/basis_set.py

```diff
@@ -87,23 +87,29 @@ def atom_basis_data(name, atom):
     >>> print(params)
     [((0, 0, 0), [3.425250914, 0.6239137298, 0.168855404], [0.1543289673, 0.5353281423, 0.4446345422])]
     """
-    basis_sets = {"sto-3g": STO3G}
+    basis_sets = {"sto-3g": STO3G, "6-31g": POPLE631G, "STO-3G": STO3G, "6-31G": POPLE631G}
 
     s = [(0, 0, 0)]
     p = [(1, 0, 0), (0, 1, 0), (0, 0, 1)]  # for px, py, pz, respectively
 
     basis = basis_sets[name][atom]
     params = []
+    sp_count = 0
     for i, j in enumerate(basis["orbitals"]):
         if j == "S":
             params.append((s[0], basis["exponents"][i], basis["coefficients"][i]))
-        elif j == "SP":
+        if j == "SP":
             for term in j:
                 if term == "S":
-                    params.append((s[0], basis["exponents"][i], basis["coefficients"][i]))
+                    params.append(
+                        (s[0], basis["exponents"][i], basis["coefficients"][i + sp_count])
+                    )
                 if term == "P":
                     for l in p:
-                        params.append((l, basis["exponents"][i], basis["coefficients"][i + 1]))
+                        params.append(
+                            (l, basis["exponents"][i], basis["coefficients"][i + sp_count + 1])
+                        )
+            sp_count += 1
```

Comment:
I think this can be cleaned up a little bit
```
>>> if j == "SP":
...    params.append(
        (s[0], basis["exponents"][i], basis["coefficients"][i + sp_count])
       )
...    for l in p:
...        params.append(
            (l, basis["exponents"][i], basis["coefficients"][i + sp_count + 1])
           )
```

Reply:
The double if statements are not required since we know the j is "SP", each if block is only called once, so we can just call them one after another. Unless I am missing something ?

Reply:
That is correct, we will need more `if`s in the future, for  P , D , ... orbitals, and using SP could be a bit misleading since we do not really have SP orbitals, so I thought it is better to have separate `if` blocks now.